### ISSUE TITLE
URL's that are similar but not equal are both marked as active

### ISSUE THREAD

**Issue by 24hoursmedia-craftcms** [verbb/navigation#69]
### Description

When I add items with similar url's that look alike, they are both marked as 'active'.

### Steps to reproduce

1. add an entry that would resolve to uri '/newsletter'
2. add an entry that would resolve to uri '/news'
3. navigate to '/newslettter' page
4. in menu or menus, both menu entries are marked as 'active'

I have this on multiple sites, for example on
https://www.vrijescholen.nl/nieuwsbrief both menu item 'nieuws' and 'nieuwsbrief' are active

### Additional info

- Plugin version: 1.0.18
- Craft version: Craft Solo 3.1.7

**Comment by prometeusweb** [comment]
I confirm the bug, I have entries with uri 'ricarica' and 'ricarica-oom' and both are marked as active

```  
{% nav node in nodes %}
    <li class="{{ node.active ? 'active' }}">  
        <a href="{{ node.url }}">{{ node.title }}</a>
        {% if node.active %}
            {% if node.hasDescendants %}
                <ul class="square my-1 pl-4">
                    {% set children = node.children.all() %}
                    {% for subnode in children %}
                        <li class="{{ subnode.active ? 'active' }}">{{ subnode.link }}</li>
                    {% endfor %}
                </ul>
            {% endif %}
        {% endif %}
    </li>
{% endnav %}
```

### Additional info
* Plugin version: 1.0.18
* Craft version: Craft Solo 3.1.7

**Comment by engram-design** [comment]
Fixed in [1.1.0](https://github.com/verbb/navigation/releases/tag/1.1.0)
